andela/mrm-mobile
# #159728988 Filter rooms by capacity

### What does this PR do?
Allows the user to filter through all rooms by capacity.

### Description of Task to be completed.
[x] User can filter rooms by capacity.
[x] Filtering is responsive.
[x] Test the filtering functionality.

### How should this be manually tested?
1. Set up the project locally as detailed in the README.
2. Run the app using an emulator.
3. Log in to mrm and set up a room.
4. In the room activity, click ``` find rooms ```.
5. Check out the capacity filter at the top.

### What are the relevant pivotal tracker stories?
#### 159728988
[A user can find a room by using the filter  capacity](https://www.pivotaltracker.com/story/show/159728988)

### Screenshots

![ezgif com-optimize](https://user-images.githubusercontent.com/33759757/44868274-0b9feb80-ac93-11e8-8786-21ba0e201c47.gif)
